src/Facade/Services/ImportBookFacadeService.cs

```diff
@@ -22,12 +25,135 @@ protected override ImportBook CreateFromResource(ImportBookResource resource)
 
         protected override void UpdateFromResource(ImportBook entity, ImportBookResource updateResource)
         {
-            throw new NotImplementedException();
+            entity.DateCreated = DateTime.Parse(updateResource.DateCreated);
+            entity.ParcelNumber = updateResource.ParcelNumber;
+            entity.SupplierId = updateResource.SupplierId;
+            entity.ForeignCurrency = updateResource.ForeignCurrency;
+            entity.Currency = updateResource.Currency;
+            entity.CarrierId = updateResource.CarrierId;
+            entity.OldArrivalPort = updateResource.OldArrivalPort;
+            entity.FlightNumber = updateResource.FlightNumber;
+            entity.TransportId = updateResource.TransportId;
+            entity.TransportBillNumber = updateResource.TransportBillNumber;
```

Comment:
i'd probably move all this into the domain service myself even though it's a pain. Either way definitely needs a test.